installing_on_kubernetes/_topics/external_postgres.md

```diff
@@ -0,0 +1,14 @@
+## External postgresql
+
+Running with an external Postgres server is an option, if you want the default internal Postgres you can skip this step. Additionally, if you want to secure the connection, you need to include the optional parameters `sslmode=verify-full` and `rootcertificate` when creating the secret. To do this, manually create the secret substituting your values before creating the CR.
```

Comment:
Running with an external Postgres server is an option, if you want the default internal Postgres you can skip this step. Additionally, if you want to secure the connection, you need to include the optional parameters `sslmode=verify-full` and `rootcertificate` when you create the secret. To do this, manually create the secret and substitute your values before you create the CR.

Reply:
Done